src/components/Profile/Profile.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState, useEffect } from 'react';
+
+import {
+  Container,
+  Grid,
+} from "@material-ui/core";
+
+import { get, post } from '../../requests';
+
+import SolutionCard from '../SolutionCard/SolutionCard';
+
+const Profile = ({update, setUpdate, userId, setUserId}) => {
+  const [userSolutions, setUserSolutions] = useState([]);
```

Comment:
Let's change it to `profileSolutions`. Just in case if in future we pass this somewhere, we know exactly where it comes from.